src/cargo/core/compiler/custom_build.rs

```diff
@@ -416,6 +416,7 @@ fn build_work(build_runner: &mut BuildRunner<'_, '_>, unit: &Unit) -> CargoResul
                 let values = cfg_map.entry(k.as_str()).or_default();
                 values.push(v.as_str());
             }
+            Cfg::Version(..) => todo!(),
```

Comment:
Should we have this behind an unstable gate or insta-stabilize once the stabilization report for the rustc side is done?